features/step_definitions/networking.rb

```diff
@@ -1172,4 +1172,36 @@
       @result = admin.cli_exec(:patch, resource: "network.operator", resource_name: "cluster", p: "{\"spec\":{\"defaultNetwork\":{\"ovnKubernetesConfig\":{\"mtu\": null}}}}", type: "merge")
       raise "Failed to clear mtu field from CNO" unless @result[:success]
   }
-end
\ No newline at end of file
+end
+
+Given /^I get egress data file directory to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_egress_directory" unless cb_name
+  network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
+  network_type = network_operator.network_type(user: admin)
+  case network_type
+  when "OVNKubernetes"
+    cb[cb_name]="ovn-egressfirewall"
+  when "OpenShiftSDN"
+    cb[cb_name]="egressnetworkpolicy"
+  else
+    raise "unknown network_type"
+  end
+  logger.info "The egressfirewall fie directory path is stored to the #{cb_name} clipboard."
+end
+
+Given /^I get egress type to the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
Generally, we use get ... from ..., get ... to ... looks strange.
Perhaps `Given /^I save egress type to the#{OPT_SYM} clipboard$/ do | cb_name |` or 
`Given /^I get egress type and save to the#{OPT_SYM} clipboard$/ do | cb_name |`

Reply:
@liangxia Thanks, fixed it.